hazelcast/src/test/java/com/hazelcast/client/test/Student.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This class is for Non-java clients. Please do not remove or modify.
+ */
+
+package com.hazelcast.client.test;
+
+import com.hazelcast.nio.serialization.Portable;
+import com.hazelcast.nio.serialization.PortableReader;
+import com.hazelcast.nio.serialization.PortableWriter;
+
+import java.io.IOException;
+
+public class Student implements Portable {
+    private long age;
+    private float height;
+
+    @Override
+    public int getFactoryId() {
+        return 666;
```

Comment:
Interesting choice of a number.

Reply:
Can you use the field `com.hazelcast.client.test.PortableFactory#FACTORY_ID` directly instead?